app/serializers/current_user_serializer.rb

```diff
@@ -312,7 +313,15 @@ def has_topic_draft
     true
   end
 
+  def draft_count
+    Draft.stream({ user: object, all: true }).count
+  end
+
   def include_has_topic_draft?
     Draft.has_topic_draft(object)
   end
+
+  def include_draft_count?
+    Draft.has_topic_draft(object)
```

Comment:
Since both `include_has_topic_draft?` and `include_draft_count?` do the same request, any chances this could be cached so we do the request (in `Draft.has_topic_draft`) once?